string ble_status
uint64 anchor_system_time
uint64 anchor_received_time
float64 firstPath_power
float64 aoa
float64 distance
uint64 hpc_system_time
